internal/agentdeployer/kubernetes.go

```diff
@@ -39,11 +38,10 @@ type KubernetesAgentDeployer struct {
 }
 
 type KubernetesAgentDeployerOptions struct {
-	Profile        *profile.Profile
-	DefinitionsDir string
-	StackVersion   string
-	PolicyName     string
-	DataStream     string
+	Profile      *profile.Profile
+	StackVersion string
+	PolicyName   string
+	DataStream   string
```

Comment:
If it is preferred, I can create a separated PR to remove `definitionsDir` from this code.

Reply:
Moved these changes to a new PR #1769